### ISSUE TITLE
Function A22

### ISSUE THREAD

**Issue by BananaBonanza** [louvill/AAE535dPCalc#12]
### Give a brief description of what the task will accomplish
We need to create the outline of function A22 that will be further updated as more work is done. This needs to take in fluid and component parameters and output a full array that could include all calculations (Reynolds, friction factor, ect).

### Describe the solution you'd like
***THIS IS NOT THE ISSUE FOR CALCULATING THESE PARAMETERS!*** This is just creating the actual .py file with a function within it that inputs the array from [issue 4](https://github.com/louvill/AAE535dPCalc/issues/4) and output information that will eventually be coded.

### Describe alternatives you've considered
See the resolution of [issue 7](https://github.com/louvill/AAE535dPCalc/issues/7) for alternative data structure ideas.

### Additional context
Basically we want it to look like this:
```python
def A22Function(input):
  reynolds = False
  friction = False
  Kt = False
  output = append.input([reynolds,friction,Kt])
  return(output)
```
So no actual computations yet, just the skeleton.
